gtsfm/configs/default_lund_door_set1_config.yaml

```diff
@@ -12,7 +12,7 @@ SceneOptimizer:
 
   two_view_estimator:
     _target_: gtsfm.two_view_estimator.TwoViewEstimator
-    corr_metric_dist_threshold: 0.1
+    corr_metric_dist_threshold: 0.5
```

Comment:
Thanks for the updates, Ayush. Maybe this can be 
```python
corr_metric_dist_threshold: 0.5 # in px
```

Reply:
Thanks!